src/navigator-card-body.jsx

```diff
@@ -280,7 +287,24 @@ export const NavigatorCardBody = ({
                 <Spinner />
             </Flex>
         );
-    if (isGrid) {
+    const clearFilter = () => {
+        setCurrentFilter("");
+    };
+    if (sortedFiles.length === 0) {
+        return (
+            <EmptyState>
+                <EmptyStateHeader
+                  titleText="No results found" headingLevel="h4"
```

Comment:
Please make this translatable:

```suggestion
                  titleText={_("No results found")} headingLevel="h4"
```